### PR TITLE
Fix wl team command

### PR DESCRIPTION
**Pull request by Pablete1234**

### CODE REVIEW

File: src/main/java/in/twizmwaz/cardinal/command/WhitelistCommands.java [hunk 1]
```diff
@@ -151,21 +151,17 @@ public static void kick(final CommandContext args, final CommandSender sender) t
...
-        }
-        msg = msg.trim();
+        String msg = args.getJoinedStrings(0).toLowerCase();
         if (Teams.getTeamByName(msg) == null) {
             throw new CommandException(new LocalizedChatMessage(ChatConstant.ERROR_NO_TEAM_MATCH).getMessage(ChatUtil.getLocale(sender)));
         }
         for (Player player : Bukkit.getOnlinePlayers()) {
             Optional<TeamModule> team = Teams.getTeamByPlayer(player);
-            if (team.isPresent() && team.get().getName().startsWith(msg)) {
+            if (team.isPresent() && team.get().getName().toLowerCase().startsWith(msg)) {
```
Comment (iEli2tyree011) [inline src/main/java/in/twizmwaz/cardinal/command/WhitelistCommands.java]: This is still broken, we should be comparing the teams, not the teams' names

**Comment by Pablete1234** [comment]
@iEli2tyree011 what if you have "Team red" and "Team blue", and you do /wl team team, both will get whitelisted, as both start with "team", i think this is intended behaviour

**Comment by TiTo-418** [comment]
https://github.com/twizmwazin/CardinalPGM/pull/794

**Comment by Pablete1234** [comment]
@twizmwazin opinions? i think this is a good enough fix, it will wl anyone that's on a team that starts with what you type

File: src/main/java/in/twizmwaz/cardinal/command/WhitelistCommands.java [hunk 2]
```diff
@@ -151,21 +151,17 @@ public static void kick(final CommandContext args, final CommandSender sender) t
         ChatUtil.getGlobalChannel().sendLocalizedMessage(new UnlocalizedChatMessage(ChatColor.RED + "{0}", ChatConstant.GENERIC_KICKED_NOT_WHITELISTED.asMessage()));
     }
 
-    @Command(aliases = {"team"}, desc = "Adds everyone on a team to the whitelist.", max = 1, min = 1)
+    @Command(aliases = {"team"}, desc = "Adds everyone on a team to the whitelist.", min = 1)
     @CommandPermissions("cardinal.whitelist.team")
     public static void team(final CommandContext args, final CommandSender sender) throws CommandException {
         int count = 0;
-        String msg = "";
-        for (int i = 2; i < args.argsLength(); i++) {
```
Comment (dentmaged) [inline src/main/java/in/twizmwaz/cardinal/command/WhitelistCommands.java]: ```msg = args.JoinedStrings(0);```

Comment (Pablete1234) [inline src/main/java/in/twizmwaz/cardinal/command/WhitelistCommands.java]: but sending it to lower case also helps

Comment (dentmaged) [inline src/main/java/in/twizmwaz/cardinal/command/WhitelistCommands.java]: How does it "help"?

Comment (Pablete1234) [inline src/main/java/in/twizmwaz/cardinal/command/WhitelistCommands.java]: Team is called "Red Team" and you type /wl team red

Comment (dentmaged) [inline src/main/java/in/twizmwaz/cardinal/command/WhitelistCommands.java]: And you should get the team specified in the args with ```Teams.getTeamByName(msg);```

Comment (Pablete1234) [inline src/main/java/in/twizmwaz/cardinal/command/WhitelistCommands.java]: @dentmaged as i said  b4, if there is a team called "Team Red" and a team called "Team Blue", /wl team team should wl both, not pick the first one (alphabetically) and wl them

**Comment by Pablete1234** [comment]
@iEli2tyree011 @twizmwazin i've redone this, if there are 2 teams called "Team Red" and "Team Blue" and you /wl team team, it will throw an error "Multiple teams matched querry : Team Red, Team Blue."

**Comment by Pablete1234** [comment]
@iEli2tyree011 @twizmwazin @CaptainElliott needs testing tag? some ppl should test this and show the results, IMO it's ready to merge

**Comment by TiTo-418** [comment]
All works fine

**Comment by twizmwazin** [comment]
I see some formatting that needs to be fixed.  Also, I don't think we should be removing spaces when comparing team names to the user input.

**Comment by Pablete1234** [comment]
@twizmwazin fixed 2 missing spaces (in formatting), and added the "replace(" ", "")" to the team name (missing part).
and yes, i think we should compare teams without spaces, because it makes the system more user-friendly, the same way you remove capital letters so that you don't have to writte it exactly as it is in the team name, i think getting rid of spaces also helps in that

**Comment by twizmwazin** [comment]
Rethinking over it you are probably right on the spaces issue.

**Comment by twizmwazin** [comment]
merged

**Pull request closed without merging**